pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -226,6 +226,9 @@ type Inode struct {
 
 	// imutex serializes changes to a inode.
 	imutex sync.Mutex
+
+	// link is result of following a symbolic link.
```

Comment:
"result" -> "resolved path"?

Better to explicit say it's a cache. If so, please add the "FUSE_CACHE_SYMLINKS" flag to fuse init flags

Reply:
Migrated to #3675